src/styles/styles.js

```diff
@@ -1379,7 +1379,7 @@ const styles = {
         fontFamily: fontFamily.GTA,
         fontSize: variables.fontSizeNormal,
         borderWidth: 0,
-        borderRadius: 0,
+        borderRadius: 3,
```

Comment:
Why the value 3? I saw your proposal but it didn't explain why.

Reply:
Without `borderRadius: 3`
<img width="452" alt="Screenshot 2022-07-12 at 10 26 15 PM" src="https://user-images.githubusercontent.com/83179295/178549835-a9dc7449-9d12-4bf3-a34d-0d86c4b92845.png">

With `borderRadius: 3`
<img width="452" alt="Screenshot 2022-07-12 at 10 28 03 PM" src="https://user-images.githubusercontent.com/83179295/178550012-b04bdc47-d8b9-4b7b-8840-3ae8a02d30b9.png">

Reply:
Maybe just do `borderRadius: 4` to match the spacing in increments of 4 we have all over the app? NAB

Reply:
Thanks for the screens. I agree. It should be multiple of 4. Also, use a variable from theme/default. I was going to test it on my computer before proposing this change.